### ISSUE TITLE
Folder deletion is not possible from a cloned GitHub repository

### ISSUE THREAD

**Issue by camok** [utoronto-2i2c/jupyterhub-deploy#73]
A small issue has arisen when deleting a folder cloned from a GitHub repository via nbgitpuller. The JupyterHub does not display all of the hidden files in the directory. Downloading the "empty" folder reveals many ".gitattributes" files and two hidden folders. Normally you could just delete these from a command shell but the means of doing that through the JupyterHub are not apparent. 

Is there a way for teachers/students to completely delete folders when they may contain hidden files?

Thanks!

**Comment by elb0** [comment]
I have been able to delete nbgitpuller-acquired folders through the Files pane in RStudio. Not sure if this makes sense for your/your students' workflows, but it is something that *works*.

~ Liza from the Dept of Stat Sci

**Comment by yuvipanda** [comment]
Thank you for opening this, @camok! Let me try rephrase to make sure I understand the issue.

1. Student has clicked an nbgitpuller link, and that repo is now in their home directory. 
2. They want to delete the directory once they are done with it
3. However, since the directory has some hidden files, this isn't possible from the notebook interface. 

Is this correct?

For command shell, you can access it from 'New -> Terminal' once you login to jupyter.utoronto.ca. That should give you a fully functional web shell that can execute bash commands.

Also TIL that RStudio handles hidden files better! Thanks, @elb0!

**Comment by camok** [comment]
Yes @yuvipanda that description is correct. Also thanks to you and @elb0 for the solutions. I also found that you can run a desktop (New -> Desktop) and open up a terminal in there to get rid of it. I think I found the longest route around to deleting the files. I think perhaps the nature of the Jupyter Notebook interface just prevents these kind of actions. Glad we can accomplish them through the other available web interfaces!

**Comment by yuvipanda** [comment]
@camok glad you could make it work :D Have you tried the web based terminal?

**Comment by camok** [comment]
@yuvipanda Yes I tried the terminal as well and it works like a charm. I'll advise my students to do that if they run into any issues when we eventually update the repo. Thanks!
